### PR TITLE
Fix rbac tests, make install experience better

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: tests/integration/scale_test.go [hunk 1]
```diff
@@ -37,11 +37,11 @@ func scaleTests(t *testing.T, when spec.G, it spec.S) {
 			assert.Equal(t, 0, service.GetAvailableReplicas())
 			assert.Equal(t, 0, service.GetScale())
 		})
-		it("should scale to 11", func() {
+		it("should scale to 2", func() {
 			assert.Equal(t, 1, service.GetAvailableReplicas())
-			service.Scale(11)
-			assert.Equal(t, 11, service.GetAvailableReplicas())
-			assert.Equal(t, 11, service.GetScale())
```
Comment (cbron) [inline tests/integration/scale_test.go]: Is this temporary ? Or for some reason ?

Comment (StrongMonkey) [reply tests/integration/scale_test.go]: @cbron  I don't want to launch 11 pods, I think 2 pods should be enough to test the scale functionality though...

Comment (StrongMonkey) [reply tests/integration/scale_test.go]: Launch 11 pods make tests pretty slow...

Comment (cbron) [reply tests/integration/scale_test.go]: If thats the case then @rancher-max can you move the 11 scale test to validation.

**Pull request merged**